examples/userapi/05_conditional_dimension.ipynb

```diff
@@ -71,7 +71,7 @@
      "name": "stderr",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.    op0 = Operator([eqi, eq0], opt=('advanced', {'par-collapse-ncores': 1000}))
</pre><p>Possibly add a comment line to explain the opt kwargs?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1820/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
Not strictly needed, maybe I can just drop.